docs/events/indicators-of-compromise.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[indicators-of-compromise]]
+= Indicators of compromise
+
+The Intelligence page collects data from enabled threat intelligence feeds and provides you with a centralized view of indicators. This topic offers definitions of threat intelligence and indicators, helps you get started using the Intelligence page, and explains how to work with indicators.
+
+[role="screenshot"]
+image::images/indicators-table.png[Shows the Indicators table on the Intelligence page]
+
+[discrete]
+[[ti-indicators]]
+== Threat intelligence and indicators
+If you are unfamiliar with threat intelligence, think of it as a research function that helps an organization make informed decisions when taking actions to protect itself. An organization's security operation center (SOC) team is usually responsible for managing threat intelligence. A SOC team's goals are to understand current and emerging threats so they can recommend actions that will prevent, stop, and mitigate threats to an organization.
```

Comment:
```suggestion
If you are unfamiliar with threat intelligence, think of it as research that helps an organization make informed security decisions. An organization's security operations center (SOC) team is usually responsible for managing threat intelligence. A SOC team's goals are to understand current and emerging threats so they can prevent, stop, and mitigate security threats.
```

Reply:
Thanks for these suggestions, @joepeeples! I agree that the target audience for these docs probably already understands what threat intelligence (TI) and IoCs are. In fact, I debated explaining the terms here or adding them our [terms glossary](https://www.elastic.co/guide/en/security/master/detection-engine-overview.html#det-engine-terminology) and linking to that section. I mainly wanted to see if this background info flowed well with the rest of the topic and provided the necessary context for understanding some of the more jargony terms that are used in the instructions like "threat intelligence", "threat intelligence feeds" (I probably need to add this to the terms glossary) and "indicators" (which I believe is a spelt-out abbreviated form of IoCs?). Anyways, I don't have strong feelings about keeping the content here or moving it elsewhere if everyone thinks the content is more appropriate somewhere else.

Reply:
@nastasha-solomon Thanks for the extra background. Like I said, I think this is still good content in general, and it does accomplish your goal of providing context for the terms. But we can present this information without making it sound like readers are learning about threat intel for the first time — even if they are!